x-pack/plugins/security_solution/public/common/components/auto_download/auto_download.tsx

```diff
@@ -23,7 +23,9 @@ export const AutoDownload: React.FC<AutoDownloadProps> = ({ blob, name, onDownlo
 
   useEffect(() => {
     if (blob && anchorRef?.current) {
+      // @ts-expect-error
       if (typeof window.navigator.msSaveOrOpenBlob === 'function') {
```

Comment:
Unrelated to the PR: we should probably not ship this code anymore since it's an API that was just supported by Internet Explorer. (takes a mental note to create a follow-up PR)